<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditor</class>
 <widget class="QMainWindow" name="ImageEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionRgbFilter"/>
    <addaction name="actionDrawSquare"/>
    <addaction name="actionSwapAreas"/>
    <addaction name="actionReplaceCommonColor"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Reference</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionToolsDescription"/>
    <addaction name="separator"/>
    <addaction name="actionAboutApplication"/>
   </widget>
   <widget class="QMenu" name="menu_Image">
    <property name="title">
     <string>&amp;Image</string>
    </property>
    <addaction name="actionProperties"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Image"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRgbFilter"/>
   <addaction name="separator"/>
   <addaction name="actionDrawSquare"/>
   <addaction name="separator"/>
   <addaction name="actionSwapAreas"/>
   <addaction name="separator"/>
   <addaction name="actionReplaceCommonColor"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionRgbFilter">
   <property name="text">
    <string>&amp;RGB filter</string>
   </property>
   <property name="iconText">
    <string>&amp;RGB filter</string>
   </property>
  </action>
  <action name="actionDrawSquare">
   <property name="text">
    <string>&amp;Draw square</string>
   </property>
   <property name="iconText">
    <string>&amp;Draw square</string>
   </property>
  </action>
  <action name="actionSwapAreas">
   <property name="text">
    <string>&amp;Swap areas</string>
   </property>
   <property name="iconText">
    <string>&amp;Swap areas</string>
   </property>
  </action>
  <action name="actionReplaceCommonColor">
   <property name="text">
    <string>Replace &amp;common color</string>
   </property>
   <property name="iconText">
    <string>Replace &amp;common color</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actionImageProperties">
   <property name="text">
    <string>Image properties</string>
   </property>
  </action>
  <action name="actionAboutApplication">
   <property name="text">
    <string>About application</string>
   </property>
  </action>
  <action name="action_Update_image">
   <property name="text">
    <string>Update image</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="actionAction_description">
   <property name="text">
    <string>Actions description</string>
   </property>
  </action>
  <action name="actionAboutAuthor">
   <property name="text">
    <string>About author</string>
   </property>
  </action>
  <action name="actionAboutRgbFilter">
   <property name="text">
    <string>RGB filter</string>
   </property>
  </action>
  <action name="actionAboutDrawSquare">
   <property name="text">
    <string>Draw square</string>
   </property>
  </action>
  <action name="actionAboutSwapAreas">
   <property name="text">
    <string>Swap areas</string>
   </property>
  </action>
  <action name="actionAboutReplaceCommonColor">
   <property name="text">
    <string>Replace common color</string>
   </property>
  </action>
  <action name="actionToolsDescription">
   <property name="text">
    <string>&amp;Tools description</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
